test/markdown_renderers.js

```diff
@@ -28,4 +30,22 @@ describe('apifyMarked custom renderer works', () => {
         expect(renderedTitle).to.equal(`
             <h3 id="welcome-title-id"><a href="#welcome-title-id"></a>Welcome to Apify</h3>`);
     });
+
+    it('replaces relative URLs from GitHub repos with absolute URLs in links', () => {
+        const href = './src/foo/bar';
+        const text = 'link to bar';
+        const repoUrl = `https://github.com/${repoFullName}`;
+        const renderedLink = customLinkRenderer(href, text, repoUrl, branchName);
+
+        expect(renderedLink).to.equal('<a href=https://github.com/apify/actor-test-url/tree/main/src/foo/bar rel="nofollow noreferrer noopener">link to bar</a>')
+    })
+
+    it('replaces relative URLs in images from GitHub repos with absolute URLs pointing to raw files', () => {
+        const href = './src/foo/bar.jpeg'
+        const text = 'link to bar.jpeg';
+        const repoUrl = `https://gitlab.com/${repoFullName}`;
+        const renderedLink = customImageRenderer(href, text, repoUrl, branchName);
+
+        expect(renderedLink).to.equal(`<img src=https://gitlab.com/apify/actor-test-url/-/raw/main/src/foo/bar.jpeg alt=link to bar.jpeg />`)
```

Comment:
href=https://github.com/apify/actor-test-url/tree/main/src/foo/bar

Reply:
Here, I'm testing if the gitlab conversion works, should it be gihub instead?